djangocms_versioning/templates/djangocms_versioning/admin/compare.html

```diff
@@ -13,23 +26,23 @@
             "v2_url": "{{ v2_preview_url }}",
         {% endif %}
         {% if v2 %}
-        "v2_description": "{{ v2 }} {{ v2.created }}",
+            "v2_description": "{{ v2_description }}",
```

Comment:
It's disappointing to see that when this has been changed that no tests have failed, including the fact that we are using the version number rather id now. 

We need a test that checks that the correct values are rendered in the compare view, this will help us refactor the compare view in future too because we will be checking that the values are rendered correctly. Any change to this logic in future should fail a test.

Reply:
Added test for the changes